med-web-app-front/src/services/attachment.service.js

```diff
@@ -13,22 +13,24 @@ class AttachmentService {
         return axios.get(API_URL + 'test/' + username, { headers: authHeader() });
     }
 
-    uploadAttachment(file, fileName, isDicom, onUploadProgress) {
+    uploadAttachment(file, fileName, isDicom, UID, onUploadProgress) {
         console.log("Got file to upload");
+        console.log(UID);

```

Comment:
Codacy has a fix for the issue: [Replace `········console.log(UID);␍` with `····console.log(UID);`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534713)
```suggestion
    console.log(UID);
```